api/proto/adscert.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package api;
+
+option go_package = "/cmd/internal/api";
+
+message AuthenticatedConnectionSignatureParams {
+    string invocationHostname = 1;
```

Comment:
please use underscore format, as that will automatically get converted by the code generators:

string invocation_hostname = 1;

Reply:
got it, fixed.